src/css/object/component/job-search.scss

```
@@ -0,0 +1,85 @@
+@use '../../object/utility/utility.scss';
+
+.c-jop-search {
```

タイポかなと思うので修正をお願いします！

```suggestion
.c-job-search {
```